akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/ActorLoggingSpec.scala

```diff
@@ -291,12 +238,13 @@ class ActorLoggingSpec extends ScalaTestWithActorTestKit("""
 
     "provide the MDC values in the log" in {
       val behaviors = Behaviors.withMdc[Protocol](
-        Map("static" -> 1),
+        Map("static" -> "1"),
         // FIXME why u no infer the type here Scala??
+        //TODO review that change from Map[String,Any] to Map[String,String] is viable
```

Comment:
I see, so it only has support for strings? might be a good limitation to avoid putting mutable things in there

Reply:
Only strings, yes. I agree. I think it would be more robust and also simpler (at least to maintain)